charts/tempo/templates/statefulset.yaml

```diff
@@ -17,7 +17,7 @@ spec:
   selector:
     matchLabels:
       {{- include "tempo.selectorLabels" . | nindent 6 }}
-  serviceName: {{ template "tempo.fullname" . }}-headless
+  serviceName: {{ template "tempo.fullname" . }}
```

Comment:
@dastrobu 
thanks! I have changed the suffix to be consistent with headless.
I just want to confirm one point: do you think the cluster ip value needs to be changed to default to None?
ref: https://github.com/grafana/helm-charts/blob/main/charts/tempo/templates/service.yaml#L16

Reply:
@tozastation note that I am not a maintainer of Grafana Tempo. I just ran into the same issue looking for a fix. 

Personally, I'd keep the service name (to avoid a breaking change on updating the chart for users). 

There are two potential alternatives to fix this from my point of view: 
1. add a headless service called `{{ template "tempo.fullname" . }}-headless` but keep the original service in addition. The headless service would then be used for [member list joining](https://grafana.com/docs/tempo/latest/configuration/#memberlist), the other one for pushing traces from applications. 
2. change the current service to be headless (`clusterIP: None`) and refer to it from the stateful set. I am not sure what implications it will have on clients pushing traces if the services is headless - as they will see multiple DNS records for the service. 

I think someone from the maintainers should comment on the direction to go.